src/sentry/static/sentry/app/styles/space.tsx

```diff
@@ -8,6 +8,8 @@ const space = (size: number): string => {
       return '2px';
     case 0.5:
       return '4px';
+    case 0.75:
```

Comment:
:< is it better off to just one-off `6px` instead of adding a new space size?

Reply:
that was my original thought, but I'm constantly trying to use `0.75`—It is intuitive that it would be there, since we also have `0.25` and `0.5`. 

I don't really mind having lots of increments at smaller sizes (as long as they are even) because at the end of the day I want us to get the spacing right, and I think it is easier to accept a few more sizes into the theme and try to be more dilligent about treating random margin and padding values as a hack or a smell, rather than be like "ok well use the theme _unless_ it is this thing or that thing."

Does that make sense at all? lol. Lmk if I am dumb.

Reply:
👍 	makes sense to me